### ISSUE TITLE
Terminal message looks wrong

### ISSUE THREAD

**Issue by xlifi** [cmooref17/ReservedItemSlotMods#24]
Solutions:
- Make it look more friendly
- Config option to customize message all text
- Config option to remove that message

**Comment by cmooref17** [comment]
My way of thinking about this, is that since I decided to make purchasing slots NOT enabled by default, many people wouldn't know it's an option they can enable. This is kinda like advertising that it is indeed an option for them in the config.

I agree, I don't exactly love the text there, but the screenshot you sent is from opening the terminal for the first time. Every time players open the terminal after that, they will not see that message anymore. (unless a mod does something to change it, idk)

But with all that said, if you have a solution that can satisfy the points I made, or if you think rewording the text would be better, share your thoughts here! Just because I made the mod does not mean that every decision/choices in this mod are perfect lol. I'm open to all suggestions! Maybe I could at least get rid of the strikethrough, and make the last part of the text more informative? I'm not sure.

**Comment by lucho947** [comment]
I agree with xlifi, it looks terrible. I understand that you want to ensure people are aware of the new option so I think it's fine if the message appears in default, but it would be nice if you added a option to remove the text in the config file. I and many people don't want things that can make the game feel "modded" and want a more polished-looking game. The Too Many Emotes mod had a similar issue and ended up removing it.
Of course that's on you, im not the mod creator, just pointing out my view on this. Thx for the great mod <3!

**Comment by kronflux** [comment]
Yeah this sadly feels too spammy, and is not aesthetically pleasing.
I think by now people are used to looking at configs, and should be aware of the available options when installing a mod. Advertising that an option exists is unnecessary.
If the option is disabled, there should be no text added to the terminal.
This should be the case with all options within all mods.

Alternatively, if you absolutely insist on keeping it there, please add a "Disable Terminal Messages" option, which you can also disable by default, so the default config still shows your message.

**Comment by xlifi** [comment]
Sorry, I missed this thread for like 9 days!
My solution would be to change the text (see "Images") and add a config option to disable the message.

<details>
<summary>Images</summary>
<p>(appears if you are the host)<br>
Buying Reserved Slots is currently disabled. You can enable it or hide the message in config.
<img src="https://github.com/cmooref17/ReservedItemSlotMods/assets/70767436/db076fa6-80d5-44e6-b7c9-968803fbdb43" alt="image">
(appears if you are not the host)<br>
Buying Reserved Slots is disabled by host. You can hide the message in config.
<img src="https://github.com/cmooref17/ReservedItemSlotMods/assets/70767436/941402f5-c260-40ab-afbc-26d8ba198f82" alt="image">
(appears if buying slots is enabled)<br>
Buying Reserved Slots is enabled. Type &quot;Reserved&quot; to purchase slots.
<img src="https://github.com/cmooref17/ReservedItemSlotMods/assets/70767436/fc4c6d10-b30f-4ad1-ad6b-9e083e8ab75a" alt="image"></p>
</details>
